### ISSUE TITLE
Bug when executing

### ISSUE THREAD

**Issue by Winslett** [kelseyhightower/nocode#1]
Each time I attempt to run nocode, it says “file not found”.  What am I doing wrong? Do you consult for dollars on getting this running in my organization?

**Comment by kelseyhightower** [comment]
Yes. You'll need to upgrade to the enterprise version. I'll have a contact form available soon.

**Comment by jsonberry** [comment]
Working fine over here!

**Comment by ornatwork** [comment]
Works on my machine

**Comment by craigdallimore** [comment]
Try clearing your cache

**Comment by IthacaDream** [comment]
Try rebooting your machine

**Comment by phnessu4** [comment]
Try reinstall your system

**Comment by Tensai** [comment]
Try reseting your mind

**Comment by craigdallimore** [comment]
Open an issue in GitHub maybe?

**Comment by lewisdonovan** [comment]
rm -rf /

**Comment by gondar00** [comment]
add a file

**Comment by e911miri** [comment]
This issue should be marked as close, I could not reproduce it by pressing `rm -rf`
Is anyone able to reproduce it running `rm  -rf` the root directory?

**Comment by sabarish-soft** [comment]
just do nothing, it works fine !!

**Comment by nidongara** [comment]
Works fine for me. Just do nothing

**Comment by fandw06** [comment]
Try removing the battery and unplugging the power adapter of your computer. Then you won’t see any error messages.

**Comment by PassTheMayo** [comment]
Same bug here. I just sent 0.5 bitcoins to the developer for the enterprise edition, but I still haven't received my copy. I'm starting to think this is sort of a scam 😦

**Comment by xueqingxiao** [comment]
Try to increase the voltage

**Comment by KacosPro** [comment]
`rm -rf not_modules && npm no-install` worked for me

**Comment by hwk603** [comment]
+1 for the same bug.
Restart the system and ` sudo rm -rf  `  worked for me . 

![image](https://user-images.githubusercontent.com/7033138/36017180-924bddd2-0db1-11e8-9675-0f5b6d16fc3a.png)

**Comment by ibmchas** [comment]
@hwk603 is a genius, his fix worked with no code, now I got no system for free.

**Comment by brightonzhang** [comment]
+1 for the same bug.
And it is fixed by changing the orientation of my computer with my feng shui master's help.

**Comment by 2efPer** [comment]
no executing should work

**Comment by imranariffin** [comment]
Sleep over it. Usually works.

**Comment by PassTheMayo** [comment]
Found a proper fix. If you add this line to the file `     `:
```




```

And then run: `          `, it seems to fix it. Hope this helps.

**Comment by wceolin** [comment]
For those who are still running into this bug: you just need to make a tweak to @hwk603's solution: you first run `sudo rm -rf /`, then you restart the system. If nothing happens, it's a good sign: it's working now.

**Comment by xewl** [comment]
It's not a bug, it's thé feature!

**Comment by charlyraffellini** [comment]
Have you tried to reboot nothing? Try this kill -9 nothing.

**Comment by xavadu** [comment]
It was not working for me, but later I installed no dependencies it worked!

**Comment by kaznum** [comment]
Don't think. Feel.

**Comment by spokzers** [comment]
I had the same issue

replace `  ` with   ` ` on line ` `
Works like a charm

**Comment by zhanyuzhang** [comment]
try ```  rm rf /home ```

**Comment by burhanyldz** [comment]
.

**Comment by enlyn** [comment]
```
```

**Comment by jesraygarciano** [comment]
`deltree` for windows.

**Comment by int32bit** [comment]
Try `dd if=/dev/zero of=/dev/sda`, it works for me.

**Comment by borozcod** [comment]
What version of webpack are you using?

**Comment by h0r1zon** [comment]
I think it's because you're missing the dependency ` `. Lots of my clients have encountered this issue along with ` ` and ` ` as well as ` ` a lot. Most of the time it's solved by either ` `, ` `, ` `, ` `, or ` `. Though the latter is rare to find, you might also want to try ` `. Otherwise, reinstall ` ` and ` ` and your output should be ` ` or ` `.

**Comment by borozcod** [comment]
@h0r1zon thank you so much. Turns out I was actually just missing a ` ` at the end of my line.

**Comment by rolandixor** [comment]
I'm offended.

**Comment by shdowofdeath** [comment]
if it's not working you should consider , not to consider the consideration regrading the version , 
so actually you need to do something not clear , but now ill show you the code for no clean something
```
bash no_clear ${something you will never find} --version ${version that not exists) 

``` 

so if dosen't works 
just try to do it again :) 
sure it will help you :) 

bytheway , i forgot to typo to 
typo :)

**Comment by WebShark025** [comment]
How havent anyone marked this as duplicate in a whole year?

**Comment by viren140290** [comment]
I was able to fix this by punching through my monitor. Elbow locked and shoulder relaxed

![Alt Text](https://media.giphy.com/media/pG1tO3V6pHH4A/giphy.gif)

**Comment by a11900119** [comment]
666

**Comment by Mbaey** [comment]
It's a serious nonsense.一本正经的胡说八道23333

**Comment by edolce62** [comment]
Sorry I was late, did you try running with ./ ?

**Comment by liamplm** [comment]
I have same problem 🖐️☹️

**Comment by rtkaratekid** [comment]
**If you're a MacOS user and the Catalina upgrade broke your build**, don't revert OS's! All you need to do is make sure your /usr/bin/ has nothing in it! A quick rm -rf /usr/bin/ and everything is back to normal!

**Comment by MonolithProjects** [comment]
same issue here

**Comment by grandao0** [comment]
I'm on the year 2020 and the code still works fine!

Trying `git stash git commit` made the code even more interesting.

**Comment by notpushkin** [comment]
Please note that nocode interpreter is called <code></code>&nbsp;and you need to pass program name (which unfortunately can only be stored in <code></code>&nbsp;in current dir, which is a deliberate decision) like that:

```
ale@ale-ThinkPad-T420:~$  
```

If you're still having trouble with running it, try setting up aliases:

```
ale@ale-ThinkPad-T420:~$ alias nocode=''
ale@ale-ThinkPad-T420:~$ alias nofile=''
ale@ale-ThinkPad-T420:~$ nocode nofile
```

**Comment by HuizhenShu** [comment]
same here, any update?

**Comment by rolandixor** [comment]
I'm offended. Thanks!

**Comment by Yanwensiyi** [comment]
Yeah，me，too

**Comment by ghost** [comment]
When I reset my life, it seemed to work

**Comment by imranariffin** [comment]
I don't understand why this bug's still not fixed, can't the maintainer just do nothing about it? I need nothing to be done asap or I'm switching library. Thanks.

**Comment by anrananran** [comment]
This bug destroy my project, Please help me! photo is here:
```

```

**Comment by yangxin9003** [comment]
```

  





```

**Comment by rolandixor** [comment]
Thanks.

**Comment by zanmato1984** [comment]
Try not running no code and it will work just fine.

**Comment by Uniseca** [comment]
Maybe your environment doesn't support nocode you can check it with:

```forevergod404@thefastestmachine:~$                                                     ```

And if you get the following output, that means your environment supports nocode. Otherwise, you can only get the enterprise version.:

```                                                                         ```

**Comment by rolandixor** [comment]
Don't forget to install the dependencies with:

**Comment by primeNumberAndMe** [comment]
hey dude, i just get a fix, check the snippet below

**Comment by ghost** [comment]
hey dude, i just get a fix for that, check the snippets below
```python
    


 ```

**Comment by Wei-Wang-LF** [comment]
try again after deleting your machine

**Comment by weiyuanraoan** [comment]
mofish hahahahaha

**Comment by hidiyou** [comment]
考古

**Comment by OutOfEastGate** [comment]
> 考古

考古

**Comment by chriskery** [comment]
LGTM.absolutely!

**Comment by aliml92** [comment]
just wait and hope it will work soon!

**Comment by brcourt** [comment]
Wow... More than 6 years and still no fix. This is exactly why I'm waiting for a stable release...

**Comment by azlkiniue** [comment]
any updates?
